DESCRIPTION

```diff
@@ -39,8 +40,10 @@ Suggests:
     ComplexUpset,
     ggplot2,
     scales,
-    curl
+    curl,
+    sf
 Remotes:
-    krassowski/complex-upset
+    krassowski/complex-upset,
+    itsleeds/osmextract
```

Comment:
Remotes is how you specify packages not on CRAN @layik. Let's see if this passes the tests, hope so!